_data/sites.json

```
@@ -15059,6 +15099,16 @@
         ]
     },
 
+    {
+        "name": "Niche",
+        "url": "https://preferences.niche.com",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```
Moving to hard since it needs a deletion request, which likely is human-aided.